projects/backend/controllers/issue.ts

```diff
@@ -33,11 +34,21 @@ export const issueController = (req: Request, res: Response) => {
         .catch(e => console.error(e))
 }
 
+const getEditionOrFallback = (editionType: string) =>
+    editionType ? editionType : 'daily-edition'
+
 export const getIssuesSummary = async (
+    editionType: string,
```

Comment:
I'd prefer `maybeEdition` here as I don't think it's describing a type.

Reply:
will change to `edition`, it's not optional

Reply:
aa it is here will do `maybeEdition`